libs/search-widget/src/components/answer/Answer.svelte

```diff
@@ -26,11 +27,11 @@
 <div class="sw-answer">
   <div class="answer-container">
     <div class="text">{@html text}</div>
-    {#if !isMobile && !hideFeedback}
+    {#if $hideSources && !isMobile && !hideFeedback}
```

Comment:
The feedback button is missing in mobile. Maybe is because of this condition ?

Reply:
oh right!
thanks!